.gitignore

```diff
@@ -189,3 +189,6 @@ _build/
 .DS_STORE
 .DS_Store
 
+
+# SOAP
+solver_cache
```

Comment:
As far as I understand, this is not the default name for the cache, but rather the one that you use in your installation. Maybe it would make sense to make it default (e.g., when one is not set by the user)?